### PR TITLE
Add getting started instructions (with makefile tweaks)

### PR DESCRIPTION
**Pull request by jasonk000**
Create specific make targets for running each of the three separate components (profiler, rmi host, and the workload).

Fill out the readme to show how to get an instance running with practical results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,86 @@ For documentation, including installing, building, and using JCoz, please see ou
...
+
+Open the [coz UI here](https://plasma-umass.org/coz/), and upload the file and review the output.
+
+## Profiling a real application
+
+You should now be in a position to profile a real application. Use the JCozCLI
+and capture some samples!
+
+Be aware for that a real sized application there will be lots of code and lots
+of experiments that coz needs to run. You should plan to keep coz running for
```
Comment (Octogonapus) [inline README.md]:
```suggestion
of experiments that JCoz needs to run. You should plan to keep JCoz running for
```

File: Makefile [hunk 2]
```diff
@@ -101,12 +101,21 @@ clean:
...
 	  -agentpath:$(BUILD_DIR)/liblagent.so \
 	  -cp $$(readlink -f ../../../target/client*dependencies.jar):. \
-	  test.TestThreadSerial &
+	  test.TestThreadSerial --fast
 
-kill:
-	pkill -9 java
+run-rmi-host:
+	java \
+	  -cp ./src/java/target/client-0.0.1-jar-with-dependencies.jar:/usr/lib/jvm/java-8-openjdk-amd64/lib/tools.jar \
```
Comment (streetster) [inline Makefile]: suggest `client-*-jar-with-dependencies.jar` so this doesnt break as/when the version gets bumped from `0.0.1`

File: README.md (hunk 1 above)
Comment (jasonk000) [reply README.md]: Thanks, updated.

File: Makefile (hunk 2 above)
Comment (jasonk000) [reply Makefile]: Thanks, updated this. Also updated to use `$JAVA_HOME` instead of a fixed jvm path.

**Pull request merged**